handler_test.go

```diff
@@ -0,0 +1,54 @@
+package lab2
+
+import (
+	"bytes"
+	"github.com/stretchr/testify/assert"
+	"strings"
+	"testing"
+)
+
+func TestCompute_Correct_1(t *testing.T) {
+	output := new(bytes.Buffer)
+	handler := ComputeHandler{
+		Input:  strings.NewReader("2 6 +"),
+		Output: output,
+	}
+	err := handler.Compute()
+	if assert.Nil(t, err) {
+		assert.Equal(t, "8", output.String())
```

Comment:
i dont think that result will be like this. remember case in "17" and "17.00000" (because of fload64)